Userland/Utilities/tree.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020, Stijn De Ridder <stijn.deridder@hotmail.com>
+ * Copyright (c) 2020-2022, Stijn De Ridder <stijn.deridder@hotmail.com>
```

Comment:
This isn't your copyright attribution for you to update, is it?

Reply:
No. I can change it with Copyright (c) 2022 Names4Noobs if that is ok.